zc_common/timezone.py

```diff
@@ -133,8 +133,16 @@ def parse(date_string, **kwargs):
     """ A wrapper around python-dateutil's parse function which ensures it always returns an aware datetime """
     from dateutil.parser import parse as datetime_parser
     from django.utils.timezone import is_aware, make_aware
+    from dateutil.tz import *
 
-    parsed = datetime_parser(date_string, **kwargs)
+    tzinfos = {'EDT': tzfile('/usr/share/zoneinfo/America/New_York'),
```

Comment:
I think these should be [`gettz`](https://labix.org/python-dateutil#head-b79630fbbda87af6d6d121737510fd7ea5aeea97) so we don't tie the library to a specific os/implementation.